parlai/core/image_featurizers.py

```diff
@@ -167,7 +170,7 @@ def img_to_ascii(self, path):
             asc.append('\n')
         return ''.join(asc)
 
-    @first_n_cache
+    # @first_n_cache
```

Comment:
did you mean to disable the cache?

Reply:
yes, as it was caching the images in GPU which led to cuda OOM errors; though now that I think about it, I could just cache the cpu version, as loading those from cache is still faster than loading from disk. But this would probably involve moving the cuda logic to the cache, rather than the load/extract methods